data/hai/unfettered side missions.txt

```diff
@@ -65,3 +65,471 @@ mission "Unfettered returning home"
+				`	"No, thanks. I'm not looking to add another ship to my fleet."`
+			`	The two Hai look disappointed. "A shame. We will probably not meet again. Farewell," one says. They signal the ship from earlier to submerge into the ocean again and walk off along the beach.`
+				goto "train back"
+			label pilot
+			action
+				set "unfettered: pilot dialog"
+			`	"Actually, it's a 'what,' not a-" starts the first one, but they are again interrupted by the second.`
+			`	"The less you know, the better for all of us, Captain."`
+				goto choice
+			label ocean
+			action
+				set "unfettered: ocean dialog"
+			`	"We found it there," the first one answers.`
+			`	"It is hard to find a drop of rain in a storm, but the drop of rain may see all," the other one adds.`
+				goto choice
+			label take
+			action
+				give ship "Anomalocaris (Venerable)" "Good Omen"
+					id "hai legacy"
+				fine 10000000
+			`	They give you access to the ship and proceed to explain the details. "It is of no real significance, but surely you understand we cannot go through standard means for this sale. We will instead present it as a fine. You'll be charged for carrying technology you had no idea was forbidden, and it will not go on your permanent record. These Korath weapons are suspicious-looking indeed."`
+			`	You quickly realize that they are not asking you for your opinion at all. In fact, you receive notice of the fine before they have even finished speaking. "We will need to make some modifications to the ship before you can use it.`
+			`	"Oh, I almost forgot: make sure you return to the spaceport to get the cargo loaded onto your ship. You should use this ship and this ship alone to make these urgent deliveries. Arriving in a different ship or with a supporting fleet will unnerve the Unfettered."`
+			choice
+				`	(Go back to the shipyard.)`
+			`	They walk away far enough so that you cannot hear them, leaving you with the access codes to the ship so you can bring it to the shipyard. With the aid of a surprisingly advanced navigation AI, you manage to bring it all the way there effortlessly despite the ship's multiple crew design. When you arrive, however, the AI shuts down without explanation, and you're forced to use a full crew complement.`
+				accept
+			label foolish
+			`	You get your weapon out and proceed to shoot on the ship.`
+			branch miss
+				random < 50
+			branch lucky?
+				random < 2
+			`	The shot reaches the ship, but the small weapon has no chance of doing anything to the heavy armor, even in its current state.`
+			`	The ship then shoots a warning shot on the ground with its Bullfrog Anti-Missiles, causing a geyser of sand to erupt on you and forcing you to drop your weapon. One of the two Hai approaches you, seemingly disappointed. "You're lucky to still be alive, but even so you are of no use to us if you think it wise to use your firearm on a spaceship. Farewell." They turn and leave without another word, and the ship flies off.`
```

Comment:
```suggestion
			`	Your shot reaches the ship, but the small weapon has no chance of doing anything to the heavy armor, even if it is somewhat worn. The ship then shoots back at you, barely missing and causing a geyser of sand to erupt at you and force you to drop your weapon. One of the two Hai then approaches you, seemingly disappointed.` 
			`	"You're lucky to still be alive, but even so you are of no use to us if you think it wise to use your firearm on a spaceship. Farewell." They turn and leave without another word, and the ship flies off.`
```
A few edits here. Combined the two indents the incidents are tied together, and there's not much narrative value of singling out the first sentence. Instead, I've indented the last part, which is more pronounced as it is the result of things. Also, it would be better not to be explicit about the type of weapon (and it is odd for a ship to fire an AM at anything that isn't a missile, since they're presumably automated).